lib/common/tests/xml/pcmk__xe_copy_attrs_test.c

```diff
@@ -58,7 +78,94 @@ copy_one(void **state)
     free_xml(target);
 }
 
+static void
+copy_multiple(void **state)
+{
+    xmlNode *src = pcmk__xe_create(NULL, "test");
+    xmlNode *target = pcmk__xe_create(NULL, "test");
+
+    pcmk__xe_set_props(src,
+                       "attr1", "value1",
+                       "attr2", "value2",
+                       "attr3", "value3",
+                       NULL);
+
+    pcmk__xe_copy_attrs(target, src, true, false);
+
+    assert_string_equal(crm_element_value(src, "attr1"),
+                        crm_element_value(target, "attr1"));
+    assert_string_equal(crm_element_value(src, "attr2"),
+                        crm_element_value(target, "attr2"));
+    assert_string_equal(crm_element_value(src, "attr3"),
+                        crm_element_value(target, "attr3"));
+
+    free_xml(src);
+    free_xml(target);
+}
+
+static void
+overwrite(void **state)
+{
+    xmlNode *src = pcmk__xe_create(NULL, "test");
+    xmlNode *target = pcmk__xe_create(NULL, "test");
+
+    // Overwrite true
```

Comment:
separate tests for overwrite and do_not_overwrite would be cleaner (similar for score_update)